src/components/Catalogue/Filters.js

```diff
@@ -42,6 +42,7 @@ const Filters = (props) => {
             }}
             max="10000"
           />
+          <p>10000</p>
         </div>
         <div class="Rating xs-gutter">
           <div class="para-sm-xs bold">Sort by Price</div>
```

Comment:
I think its a typo, but in react we have to use ``className`  instead of `class`

Reply:
changed it.